src/lib.rs

```diff
@@ -936,6 +952,45 @@ impl<'a> Face<'a> {
         }
     }
 
+    /// Returns a horizontal typo face ascender.
```

Comment:
Could you note in the docs, that a user should prefer ascender/descender methods. And those one are more low level.

Also, could you use *typographic* instead of *typo*?